Dockerfile

```diff
@@ -1,16 +1,25 @@
-FROM maven:3.5-jdk-8
+FROM maven:3.5-jdk-9-slim as build
+
+# procps is very common in build systems, and is a reasonably small package
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		git \
+	&& rm -rf /var/lib/apt/lists/*
+
 
 WORKDIR /iri
 
-RUN git clone -b master https://github.com/iotaledger/iri.git /iri/
+RUN git clone -b v1.4.1.2 https://github.com/iotaledger/iri.git /iri/ 
```

Comment:
Bummer, but not possible due to the autobuild functionality of the docker hub project https://hub.docker.com/r/bluedigits/iota-node/
which is triggered by pushes to the official iotaledger/iri project. Pointing to master ensures that the _latest_ version always points to the most recent one.